docs/search/views/index.md

```diff
@@ -1,309 +1,304 @@
-
-- **consolidationPolicy**:
-  - Type: `object`
-  - Default: `{}`
-
-The `consolidationPolicy` property has the following optional sub-properties:
-
-- **type**: Choose between `"bytes_accum"` or `"tier"`.
-  - Type: `string`
-  - Default: `"bytes_accum"`
-
-If you choose `"bytes_accum"`, the following optional sub-properties are available:
-
-- **threshold**: Define a number between `0.0` and `1.0` to act as multiplier to determine the threshold for consolidating each segment.
-  - Type: `float`
-  - Default: `0.1`
-
-	We use the following formula to set the threshold:
-
-	`t > (s + m) / a`
-
-  - `t` is the threshold value.
-  - `s` is the total bytes of the segment.
-  - `m` is the byte total of the merge candidate segments.
-  - `a` is the byte total of all segments.
-
-If you choose `"tier"`, the following optional sub-properties are available:
-
-- **segmentsMin**: Minimum number of segments considered for consolidation.
-  - Type: `integer`
-  - Default: `1`
-- **segmentsMax**: Maximum number of segments considered for consolidation.
-  - Type: `integer`
-  - Default: `10`
-- **segmentsBytesMax**: Maximum allowed size of all consolidated segments in bytes.
-  - Type: `integer`
-  - Default: `5368709120`
-- **segmentsBytesFloor**: When choosing segments for consolidation, all segments smaller than this value in bytes are automatically rounded up to this number for the purpose of consolidation selection.
-  - Type: `integer`
-  - Default: `2097152`
-
-### Write Buffers
-
-A C8Search index contains writer objects that are mapped to processed segments. You can set up a _pool_ of writers by using `writebuffer*` properties to limit memory usage. These options are immutable once a View is created.
-
-- **writebufferIdle**: Maximum number of writers cached in the pool. To disable, set `0`.
-  - Type: `integer`
-  - Default: `64`
-- **writebufferActive**: Maximum number of concurrent active writers performing a transaction. Other writers must wait until current active writers finish.
-  - Type: `integer`
-  - Default: `0`
-- **writebufferSizemax**: Maximum memory size in bytes per writer before triggering writer flush. If set `0`, the limit is removed and this option falls back on the C8DB default flush interval. Disable this option with caution.
-  - Type: `integer`
-  - Default: `33554432`
+---
+sidebar_position: 1
+title: Search Views
+slug: views
+---
+
+We provide _search views_ that enable you to perform fast and sophisticated full-text search queries for unstructured data in documents across multiple linked collections. You can filter by document attributes, sort the results by relevance, and rank results by their similarity using popular scoring algorithms.
+
+Each search view represents an inverted index that contains the search configuration and all document attributes in the linked collections. A search view index consists of multiple segments that are each treated as a standalone index.
+
```

Comment:
```suggestion

## Search View vs. Full-Text Index

```

Reply:
This section might make more sense after the following section, or even as a separate topic. Also, consider linking to the full-text index topic.